nemo/lightning/README.md

```diff
@@ -0,0 +1,11 @@
+# NeMo Lightning
+
+This directory provides custom PyTorch Lightning-compatible objects for seamlessly training NeMo 2.0 models using PTL. NeMo 2.0 models
+are implemented using [Megatron Core](https://github.com/NVIDIA/Megatron-LM/tree/main/megatron/core), and NeMo Lightning provides the bridge between higher-level object-oriented PTL APIs and lower-level Megatron APIs. 
+For detailed tutorials and documentation of NeMo 2, please refer to the NeMo 2.0 [docs](https://docs.nvidia.com/nemo-framework/user-guide/latest/nemo_2.0/index.html) (TODO: update this hyperlink).
```

Comment:
Do not forget to remove TODO internal note.

For detailed tutorials and documentation on NeMo 2.0, refer to [docs](https://docs.nvidia.com/nemo-framework/user-guide/latest/nemo_2.0/index.html).